src/uproot/behaviors/TH1.py

```diff
@@ -292,14 +309,20 @@ def to_numpy(self, flow=False, dd=False):
         else:
             return values, xedges
 
-    def to_boost(self, metadata=boost_metadata, axis_metadata=boost_axis_metadata):
+    def to_boost(self, metadata=None, axis_metadata=None):
+        if axis_metadata is None:
+            axis_metadata = boost_axis_metadata
+        if metadata is None:
+            metadata = boost_metadata
+
         boost_histogram = uproot.extras.boost_histogram()
 
         values = self.values(flow=True)
 
         sumw2 = self.member("fSumw2", none_if_missing=True)
 
-        if sumw2 is not None and len(sumw2) == self.member("fNcells"):
+        # TODO: this is a temporary fix
+        if sumw2 is not None and len(sumw2) == self.member("fNcells") and False:
```

Comment:
(I had missed this one.)

```suggestion
        # TODO: this is a temporary fix
        if sumw2 is not None and len(sumw2) == self.member("fNcells"):
```